static-assets/js/tinymce-plugins/ace/plugin.js

```diff
@@ -39,27 +39,28 @@ tinymce.PluginManager.add(PLUGIN_NAME, function (editor, url) {
       '<svg width="20px" height="20px" viewBox="0 0 25 25"><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><path d="M4,5 C4,4.71874859 4.0963532,4.4817718 4.2890625,4.2890625 C4.4817718,4.0963532 4.71874859,4 5,4 L11,4 C11.2812514,4 11.5182282,4.0963532 11.7109375,4.2890625 C11.9036468,4.4817718 12,4.71874859 12,5 C12,5.28125141 11.9036468,5.5182282 11.7109375,5.7109375 C11.5182282,5.9036468 11.2812514,6 11,6 L7.421875,6 L18,16.578125 L18,13 C18,12.7187486 18.0963532,12.4817718 18.2890625,12.2890625 C18.4817718,12.0963532 18.7187486,12 19,12 C19.2812514,12 19.5182282,12.0963532 19.7109375,12.2890625 C19.9036468,12.4817718 20,12.7187486 20,13 L20,19 C20,19.2812514 19.9036468,19.5182282 19.7109375,19.7109375 C19.5182282,19.9036468 19.2812514,20 19,20 L13,20 C12.7187486,20 12.4817718,19.9036468 12.2890625,19.7109375 C12.0963532,19.5182282 12,19.2812514 12,19 C12,18.7187486 12.0963532,18.4817718 12.2890625,18.2890625 C12.4817718,18.0963532 12.7187486,18 13,18 L16.578125,18 L6,7.421875 L6,11 C6,11.2812514 5.9036468,11.5182282 5.7109375,11.7109375 C5.5182282,11.9036468 5.28125141,12 5,12 C4.71874859,12 4.4817718,11.9036468 4.2890625,11.7109375 C4.0963532,11.5182282 4,11.2812514 4,11 L4,5 Z" fill="#000000"/></g></svg>'
   };
 
+  const onAction = (context, editor) => {
+    const isFullscreen = editor.plugins.fullscreen.isFullscreen();
+    if (isFullscreen || !inlineMode) {
```

Comment:
@rart the 4.0.x plugin has an inlineMode parameter that is needed for ICE rte control, 3.1.x doesn't need it.